configuring-sys-image-registry.html.md.erb

```diff
@@ -138,54 +135,29 @@ To upload the TAS for Kubernetes images to the private registry:
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml
+        sed -i 's/image: registry.pivotal.io/image: PRIVATE_REPOSITORY/g' relocated_images.yml
         ```
 
         Where:
-        * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your 
-        TAS for Kubernetes installation.
-        * `ORG-NAME` is your organization name, such as a Cloud Foundry Org name.
-        * `REPOSITORY` is the private registry repository to log in to.  
+        * Where `PRIVATE_REPOSITORY` is the private registry repository to log in to. Either FQDN or IP address.
+        * `PROJECT` is the project name in your private registry
+        * `IMAGE` is the image name to reallocate the images.
 <br>
     * If your client does not have the private registry CA certificates:  
 
         ```
         kbld unpackage -f <(ytt -f config/image-overrides.yml \
                          -f config/values.yml) \
                          -i /tmp/images.tar \
-                         -r registry.SYSTEM-DOMAIN/ORG-NAME/REPOSITORY \
+                         -r PRIVATE_REPOSITORY/PROJECT/IMAGE \
                          --lock-output relocated_images.yml \
                          --registry-verify-certs=false
+        sed -i 's/image: registry.pivotal.io/image: PRIVATE_REPOSITORY/g' relocated_images.yml
         ```
 
         Where:
-        * Where `SYSTEM-DOMAIN` is the system domain, such as `example.com`, that you previously selected for your 
-        TAS for Kubernetes installation.
-        * `ORG-NAME` is your organization name, such as a Cloud Foundry Org name.
-        * `REPOSITORY` is the private registry repository to log in to.  
-
-### <a id='configure-for-private-registry'></a> Customize Your Configuration to Use Your Private System Registry
-
-To customize your configuration files for your private registry: 
-
-1. Move the `config/image-overrides.yml` file to the `config-optional` directory.
```

Comment:
During "kpack package" all images are renamed to "registry.pivotal.io". 
The reallocation file created during "kpack unpackage" is based on the assumption that images are all from "registry.pivotal.io". 
Without this image-overrides, we are unable to convert the images via kbld properly as it is needed to covert to registry.pivotal.io